src/vm/loaderallocator.cpp

```diff
@@ -1005,7 +1005,8 @@ void LoaderAllocator::Init(BaseDomain *pDomain, BYTE *pExecutableHeapMemory)
                                                                       dwExecutableHeapReserveSize,
                                                                       LOADERHEAP_PROFILE_COUNTER,
                                                                       NULL,
-                                                                      TRUE /* Make heap executable */);
+                                                                      TRUE /* Make heap executable */,
+                                                                      FALSE /* Do NOT zero-initialize */);
```

Comment:
Maybe useful to describe why - that it is needed by the `UMEntryThunkCode::Poison`.